third_party/move/move-bytecode-verifier/transactional-tests/tests/ability_field_requirements/fields_conditional_abilities.v2_exp

```diff
@@ -0,0 +1,37 @@
+processed 8 tasks
```

Comment:
How does that differ from the simple .exp? There shouldn't be any difference. Is it the definition indices?